apps/test-suite/tests/NewNotifications.js

```diff
@@ -276,6 +278,73 @@ export async function test(t) {
         });
       });
     });
+
+    const onlyInteractiveDescribe = isInteractive ? t.describe : t.xdescribe;
+    onlyInteractiveDescribe('tapping on a notification', () => {
+      let subscription = null;
+      let event = null;
+
+      t.beforeEach(async () => {
+        Notifications.setNotificationHandler({
+          handleNotification: async () => ({
+            shouldShowAlert: true,
+          }),
+        });
+        subscription = Notifications.addNotificationResponseReceivedListener(anEvent => {
+          event = anEvent;
+        });
+      });
+
+      t.afterEach(() => {
+        if (subscription) {
+          subscription.remove();
+          subscription = null;
+        }
+        Notifications.setNotificationHandler(null);
+        event = null;
+      });
+
+      t.it(
+        'calls the “notification response received” listener with default action identifier',
+        async () => {
+          const secondsToTimeout = 5;
+          const shouldRun = await Promise.race([
+            askUserYesOrNo('Could you tap on a notification when it shows?'),
+            waitFor(secondsToTimeout * 1000),
+          ]);
+          if (!shouldRun) {
+            Alert.alert(
```

Comment:
(same as other PR review) -- let's log something to the console here as well to reduce the chance of writing this off as a false positive

Reply:
Done!